mixes/components/primary-track-row.js

```diff
@@ -0,0 +1,107 @@
+const React = require('react')
```

Comment:
feel free to ignore this file... it's almost entirely about getting `react-table` and `react-dnd` to play nice together.

the important part is it exposes an API `createPrimaryTrackRow({ onDrop, onDragEnter })` which is used by `PrimaryTrackTable` to generate primary track rows